entry.go

```diff
@@ -84,8 +89,9 @@ func (te *TunaEntry) Start() {
 			time.Sleep(1 * time.Second)
 			continue
 		}
-
-		tcpPorts, err := te.listenTCP(te.ListenIP, te.Service.TCP)
+		reverseBytesIn := make([]uint64, 256)
+		reverseBytesOut := make([]uint64, 256)
+		tcpPorts, err := te.listenTCP(te.ListenIP, te.Service.TCP, reverseBytesIn, reverseBytesOut)
```

Comment:
We can make `reverseBytesIn` and `reverseBytesOut` properties of `TuneEntry` just like `bytesIn`, `bytesInPaid`, `bytesOut`, `bytesOutPaid`